backend/src/kitconcept/intranet/locales/de/LC_MESSAGES/kitconcept.intranet.po

```diff
@@ -213,6 +281,14 @@ msgstr "Für authentifizierte Benutzer sichtbar, für Redakteure und Prüfer edi
 msgid "Waiting to be reviewed, not editable by the owner."
 msgstr "Wartet auf Überprüfung, kann vom Eigentümer nicht bearbeitet werden."
 
+#: kitconcept/intranet/services/feedback_contact_form.py:269
+msgid "Your feedback on our intranet content: {title}"
+msgstr "Ihr Feedback zu unseren Intranet-Inhalten: {title}"
+
+#: kitconcept/intranet/services/feedback_contact_form.py:214
+msgid "error.email"
+msgstr "Fehler.E-Mail"
```

Comment:
This does not look like it should be translated

Reply:
It is a heading for a failing mail error.

Reply:
In that case you should maybe use something better for the English version as well?